pennylane_qulacs/qulacs_device.py

```diff
@@ -181,17 +182,27 @@ def __init__(self, wires, shots=1000, analytic=True, gpu=False, **kwargs):
 
         self._circuit = QuantumCircuit(self.num_wires)
 
-    def apply(self, operations):
-        for i, op in enumerate(operations):
+    def apply(self, operations, **kwargs):
+        rotations = kwargs.get("rotations", [])
+
+        self.apply_operations(operations)
+
+        # Rotating the state for measurement in the computational basis
+        self.apply_operations(rotations)
+
+    def apply_operations(self, operations):
+        """Apply the circuit operations to the state.
+
+        This method serves as an auxiliary method to :meth:`~.QulacsDevice.apply`.
+
+        Args:
+            operations (List[pennylane.Operation]): operations to be applied
+        """
+
+        for op in operations:
             wires = op.wires
             par = op.parameters
 
```

Comment:
Isn't this done by the qnode already?

Reply:
This would be mainly to use `par` instead or writing `op.parameters` later in the function, I think. What do you mean by this being done by the qnode?

Reply:
Sorry I mean the deletion below. The qnode should check for those ops being first...